R/process_extract_alarms_telecare.R

```diff
@@ -33,10 +33,10 @@ process_extract_alarms_telecare <- function(
       .data[["record_keydate1"]],
       .data[["record_keydate2"]]
     )) %>%
-    dplyr::left_join(
-      client_lookup,
-      by = c("sending_location", "social_care_id")
-    ) %>%
+    #   dplyr::left_join(
+    #    client_lookup,
+    #   by = c("sending_location", "social_care_id")
+    # ) %>%
```

Comment:
agree

Reply:
Should I delete these commented lines or keep them?